lbrynet/lbrynet_daemon/auth/server.py

```diff
@@ -145,7 +145,7 @@ def _handle_dropped_request(self, result, d, function_name):
 
     def render(self, request):
         time_in = utils.now()
-        assert self._check_headers(request), InvalidHeaderError
+        # assert self._check_headers(request), InvalidHeaderError
```

Comment:
What's the new origin with electron, and why not update accordingly? If anything I think we should use the file system access in electron to get an authenticated api session for everything the ui does, removing the header check would be a regression on that front.